OSDOCS#6381:Adds release notes for console RFEs (#63604)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

[id="ocp-4-14-web-console"]
=== Web console

[id="ocp-4-14-administrator-perspective"]
==== Administrator Perspective
With this release, there are several updates to the *Administrator* perspective of the web console. You can:

* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
* Display and hide toolitps in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.

> 

Author: Yes thank you! I will add a link to the procedure doc once it is done.

[id="ocp-4-14-web-console"]
=== Web console

[id="ocp-4-14-administrator-perspective"]
==== Administrator Perspective

With this release, there are several updates to the *Administrator* perspective of the web console. You can:

* Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.
* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.
* Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.

[id="console-supports-installing-specific-operator-versions"]
===== Support for installing specific operator versions in the web console

> With this update, the console operator builds a set of custer `nodeOperatingSystems` types so only operators that can be installed render. You can then choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the operator will immediately update back to the latest version on the channel.

Reviewer: So the first sentence is not related to the `Support for installing specific operator versions in the web console` feature. Its rather a separate functionality, where the console-operator is gathering the OS types of each cluster nodes and based on that creates a list of supported OS types on the cluster. That list is then passed to the console, so it will only render operators, in the operator hub, which are supporting that OS type.

Author: Thanks I wasn't sure on that! I have added a separate entry.

[id="ocp-4-14-web-console"]
=== Web console

[id="ocp-4-14-administrator-perspective"]
==== Administrator Perspective

> With this release, there are several updates to the *Administrator* perspective of the web console. You can:

Reviewer 2: Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) we should avoid introducing lists with incomplete sentences, can you reword to something like this?

```suggestion
With this release, there are several updates to the *Administrator* perspective of the web console. You can now perform the following actions:
```

> * Narrow down the list of resources in a list view or search page with exact search capabilities. This will help when you have similarly named resources and fuzzy search doesn't narrow down your search.

Reviewer 2: Just a question: is fuzzy search a legit term that's used in the industry and/or our docs? Disregard if it is.

Author: Yes, it is correct only in this case.

> It is only correct to use "fuzzy" as an adjective when referring to "fuzzy searches"

From SSG https://redhat-documentation.github.io/supplementary-style-guide/#_f

> * Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and click *Share Feedback* from the drop-down list.

Reviewer 2: Suggestion for both instances of "click" to be in the present progressive tense:

```suggestion
* Provide direct feedback about features and report a bug by clicking the *Help* button on the toolbar and clicking *Share Feedback* from the drop-down list.
```

* Display and hide tooltips in the YAML editor. The tooltips will persist so you don't have to change it every time you navigate to the page.

[id="supported-os-types-cluster"]

> ===== Retrival of supported operating systems for each cluster node

Reviewer: ```suggestion
===== Operating system based filtering in Operator Hub 
```

With this update, Operators in the Operator Hub are now filtered based on the nodes operating system since cluster can contain heterogenous nodes.

[id="console-supports-installing-specific-operator-versions"]

> ===== Support for installing specific operator versions in the web console

Reviewer 2: Nit:
```suggestion
===== Support for installing specific Operator versions in the web console
```

> With this update, the `console-operator` gathers `nodeOperatingSystems` types for each cluster and creates a list of supported operating system types. That list is then passed to the console so it will only render operators in the operator hub that are supported with that operating system type.

Reviewer: lets rather avoid implementation details.
```suggestion
Operators in Operator Hub are now being filtered based on the nodes operating system, since cluster can contain heterogenous nodes.
```

> With this update, you can then choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the operator will immediately update back to the latest version on the channel.

Reviewer 2: Nits:

```suggestion
With this update, you can now choose from a list of available versions for an Operator based on the selected channel on the *OperatorHub* page in the console. Additionally, you can view the metadata for that channel and version when available. When selecting an older version, a manual approval update strategy is required, otherwise the Operator will immediately update back to the latest version on the channel.
```

Alternatively, I think you can just remove "then" without replacing it with "now".
